vcpkg/get_started/overview.md

```diff
@@ -4,12 +4,93 @@ description: Learn about how the vcpkg C/C++ package manager can help you acquir
+
+### Registries
+
+A [registry](../users/registries.md) is a catalog of ports and available versions that a vcpkg user can install. vcpkg provides a public registry of open-source libraries by default, but you can also define your own registries for custom libraries.
+
+### Asset caching
+
+[Asset caching](../users/assetcaching.md) allows vcpkg to use download-mirrors to upload and restore assets such as source code and build tools. This is useful for continuity of business, especially for open-source dependencies that have assets on third party servers. This also provides the developer protection in case upstream assets are tampered with unexpectedly.
 
 ## vcpkg compared to other package managers
 
-## Get started
+### vcpkg compared to NuGet
+
+NuGet is a .NET package manager that has often been used for C/C++ development, particularly for MSBuild solutions that also contain .NET projects. As a general rule, the Microsoft C++ team does not recommend using NuGet for C/C++ development, as NuGet has several limitations:
+
+- **Compilation flavors**. As NuGet cannot build packages from source on the fly, it is necessary to provide prebuilt binaries to match all possible Application Binary Interface (ABI) restrictions for all users. The user is responsible for building packages themselves and ensuring this is done correctly. It is also difficult to distinguish between binaries due to the lack of relevant metadata, resulting in the user having to do quirky things like putting architecture, operating system, and compiler information in the package name, without being able to enforce such constraints during package acquisition.
+- **Binary vs. source**. Very closely tied to the first point, NuGet is designed from the ground up to provide relatively small, prebuilt binaries. Due to the nature of native code, developers need to have access to the source code to ensure ABI compatibility, performance, integrity, and debuggability.
+- **Versioning limitations**. While NuGet packages are versioned, it is up to the user to decide on appropriate versions of packages to take dependencies on. The pressure is on the user to solve version conflicts between dependencies and the consuming project.
+- **No NuGet PackageReference support**. NuGet PackageReference is not supported for .vcxproj files and there is no plan to add it in the future due to technical and architectural differences between C++ and .NET MSBuild projects. This means NuGet C++ users won't benefit from features like the use of the global cache and referencing dependencies in simple MSBuild terms with access to conditional logic.
+- **Environment changes**. As NuGet C++ packages must be manually compiled and packaged, any time the host or target environment changes this work must be repeated. On the other hand, vcpkg can build packages from source for a variety of operating systems and build systems.
```

Comment:
This sounds an awful lot like "Compilation flavors". I'd replace this section with talking about transitive ABI tracking and how vcpkg can insure that when boost updates, every library using boost also rebuilds.